lualib/resty/events/broker.lua

```diff
@@ -33,6 +35,44 @@ local function is_closed(err)
             str_sub(err, -11) == "broken pipe")
 end
 
+-- broadcast to all/unique workers
+local function broadcast_events(self, unique, data)
+    local n = 0
+
+    -- if unique, schedule to a random worker
+    local idx = unique and random(1, nkeys(self._clients))
```

Comment:
If `self._client` is never empty, this code is fine.

Reply:
Yes, it is a considered issue, thank you.